main/lsp/requests/completion.cc

```diff
@@ -402,10 +403,24 @@ vector<core::LocalVariable> localsForMethod(const core::GlobalState &gs, LSPType
     return localVarFinder.result();
 }
 
+core::SymbolRef firstMethodAfterQuery(LSPTypechecker &typechecker, const core::Loc queryLoc) {
```

Comment:
Nit: Can you make it `const LSPTypechecker`? I think the below methods (`state` and `getResolved`) are marked `const`.

Reply:
I'm going to follow up with this change because I have some changes stacked locally that this will conflict with.